openedx/core/djangoapps/discussions/views.py

```diff
@@ -0,0 +1,57 @@
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from edx_rest_framework_extensions.auth.session.authentication import SessionAuthenticationAllowInactiveUser
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys import InvalidKeyError
+from rest_framework import serializers
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from openedx.core.lib.api.permissions import IsStaff
+
+from .models import DiscussionsConfiguration
+from .models import ProviderFilter
+
+
+class ConfigurationView(APIView):
+    authentication_classes = (JwtAuthentication, SessionAuthenticationAllowInactiveUser)
+    permission_classes = (IsStaff,)
```

Comment:
Currently locked down to staff.
This will probably need to open up eventually;
I can imagine `available_providers` being staff-only, but `active_provider` being available to learners too.
I figure we can open it up as needed.